kube/utils.go

```diff
@@ -0,0 +1,34 @@
+package kube
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// parsePortRange converts a port range string to a starting and an ending port number
```

Comment:
Should (IMHO) document the format of port ranges. Less mental work than deducing it from the code.
I.e. `port ranges` are `N` and `N-M` where N, M are 16bit unsigned integers, i.e. in the range (0-655535).